test/corecel/io/Repr.test.cc

```diff
@@ -73,6 +78,11 @@ TEST(ReprTest, container)
 
     std::string const strings[] = {"a", "", "special\nchars\t"};
     EXPECT_EQ("{\"a\", \"\", \"special\\nchars\\t\"}", repr_to_string(strings));
+
+    auto long_vec_str
+        = repr_to_string(std::vector<std::string>(10, std::string(75, 'x')));
+    cout << long_vec_str;
+    EXPECT_TRUE(ends_with(long_vec_str, ")\",}")) << long_vec_str;
```

Comment:
Even though it shows up with `repr_to_string`, for some reason I'm not seeing the trailing comma with `PRINT_EXPECTED`.

Reply:
I think this is because PRINT_EXPECTED writes directly to stdout, and `std::cout` must not have `tellp` (i.e. the console outputs don't keep track of how many characters they've written). Let me think about a way around this...